bifrost/app/blog/blogs/best-arize-alternatives/src.mdx

```diff
@@ -100,31 +107,61 @@ Arize Phoenix is an open-source LLM observability tool that focuses on providing
 
 Arize Phoenix excels in its evaluation capabilities and is well-suited for data scientists and ML engineers working on complex LLM projects. However, it lacks some of the developer-friendly features that Helicone offers, such as self-hosting options, user tracking, and user feedback collection. Arize Phoenix's pricing model may also be less flexible compared to Helicone's tiered approach.
 
+## Why Companies Choose Arize Phoenix Over Helicone? 
+
+- **Robust Evaluation Capabilities**: Ideal for data scientists focused on model performance.
```

Comment:
We have evals too!

Reply:
but is it the same type of eval as we do? 

https://docs.arize.com/phoenix/evaluation/llm-evals